### PR TITLE
Store blobs as [UInt8]

### PR DESCRIPTION
**Pull request by apaszke-g**

### CODE REVIEW

File: Sources/SIL/Bitcode.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ indirect enum OperandKind {
 
 indirect enum BitcodeOperand {
     case bits(_ value: Bits)
-    case blob(_ value: Bits)
+    case blob(_ value: [UInt8])
```
Comment (rxwei) [inline Sources/SIL/Bitcode.swift]: `blob([UInt8])`

File: Sources/SIL/Bitstream.swift [hunk 2]
```diff
@@ -7,6 +7,7 @@ struct Bits: Equatable, Hashable, ExpressibleByIntegerLiteral, CustomStringConve
     private var bits: [Bool]
     var description: String { String(bits.reversed().map { $0 ? "1" : "0" }) }
     var count: Int { bits.lastIndex(of: true).map { $0 + 1 } ?? 0 }
+    var isZero: Bool { bits.allSatisfy { !$0 } }
```
Comment (rxwei) [inline Sources/SIL/Bitstream.swift]: Randomly:
```suggestion
    var isZero: Bool { bits.allSatisfy(!) }
```

File: Sources/SIL/Bitstream.swift [hunk 3]
```diff
@@ -45,15 +46,6 @@ struct Bits: Equatable, Hashable, ExpressibleByIntegerLiteral, CustomStringConve
         return Int(asUInt32())
     }
 
-    func asString() -> String? {
```
Comment (rxwei) [inline Sources/SIL/Bitstream.swift]: "asXXX" is not a common pattern in Swift APIs. Usually we'd define a computed property with a noun as the name.

Comment (eaplatanios) [reply Sources/SIL/Bitstream.swift]: You can also use the [`CustomStringConvertible`](https://developer.apple.com/documentation/swift/customstringconvertible) for this particular use case with a field called `description`.

**Pull request merged**